### PR TITLE
Ignore deprecation warning emitted by the docker package

### PR DESCRIPTION
**Pull request by jankrepl**
This should be a temporary fix for https://github.com/docker/docker-py/issues/2928.

### CODE REVIEW

**Comment by FrancescoCasalegno** [comment]
For future reference — this fix is needed because `setuptools` has deprecated in https://github.com/pypa/setuptools/commit/1701579e0827317d8888c2254a17b5786b6b5246 the usage of `distutils` `Version` classes, which are still used in the `docker` Python package ([here](https://github.com/docker/docker-py/blob/a9748a8b702a3c75b46ba8c8d0490e4b8ec5ab04/docker/utils/utils.py#L8) and [here](https://github.com/docker/docker-py/blob/a9748a8b702a3c75b46ba8c8d0490e4b8ec5ab04/docker/transport/ssladapter.py#L7)).

**Review by FrancescoCasalegno — APPROVED** [review]
Awesome, thanks for fixing this.
Let's merge asap so our tests go back to running 😁

File: tox.ini [hunk 1]
```diff
@@ -138,6 +138,7 @@ commands =
 testpaths = tests
 filterwarnings =
     error
+    ignore::DeprecationWarning:docker.*:
```
Comment (Stannislav) [inline tox.ini:141]: Nice. Could we maybe add a comment line with the link to the GitHub issue? So we can directly track if it's resolved.

Comment (jankrepl) [reply tox.ini:142]: Done in 2510a38

**Review by Stannislav — APPROVED** [review]

**Pull request merged**